src/server/pbsd_main.c

```diff
@@ -299,6 +269,14 @@ int svr_unsent_qrun_req = 0;	/* Set to 1 for scheduling unsent qrun requests */
 void *jobs_idx = NULL;
 void *queues_idx = NULL;
 void *resvs_idx = NULL;
+
+void *job_attr_idx = NULL;
+void *resv_attr_idx = NULL;
+void *node_attr_idx = NULL;
+void *que_attr_idx = NULL;
+void *svr_attr_idx = NULL;
+void *sched_attr_idx = NULL;
```

Comment:
no need to initialise - they are global variables.

Reply:
curious to know: does it matter? anyway, it will same asm code right? no?

Reply:
I believe in not doing things that will be automatically done for you...
it is an optional comment

Reply:
another advantage is - uninitialized global data gets stored in the .bss section.
The nice part about this is, the .bss section data doesn't have to be included in the ELF file on disk (ie. there isn't a whole region of zeros in the file for the .bss section). Instead, the loader knows from the section headers how much to allocate for the .bss section, and simply zero it out before handing control over to your program.